Terminology (#19)
ietf-scitt/draft-birkholz-scitt-architecture: draft-birkholz-scitt-architecture.md

Artifact:

: a physical or non-physical item that is moving along the supply chain.

Statement:

: any serializable information about an Artifact. To help interpretation of Statements, they must be tagged with a media type (as specified in {{RFC6838}}).
For example, a statement may represent a Software Bill Of Materials (SBOM) that lists the ingredients of a software Artifact, or some endorsement or attestation about an Artifact.

Claim:

: an identifiable and non-repudiable Statement about an Artifact made by an Issuer. In SCITT, Claims are encoded as COSE signed objects; the payload of the COSE structure contains the Statement.

Issuer:

> : an entity that makes Claims about Artifacts in the supply chain. The Issuer may be the owner or author of the Artifact, or an independent third party such as a reviewer.

Reviewer: ```suggestion
: an entity that makes Claims about Artifacts in the supply chain. The Issuer may be the owner or author of the Artifact, or an independent third party such as a reviewer or endorser.
```

Envelope:

: the metadata added to the Statement by the Issuer to make it a Claim.
It contains the identity of the Issuer and other information to help Verifiers identify the Artifact referred in the Statement. A Claim binds the Envelope to the Statement. In COSE, the Envelope consists of protected headers.

Feed:

: an identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Registry on a Transparency Service contains a sequence of Claims about the same Artifact.

> In COSE, Feed is one header attributes in the protected header of the Envelope.

Reviewer: ```suggestion
In COSE, Feed is a dedicated header attribute in the protected header of the Envelope.
```

> Registry (or Log, or Ledger):

Reviewer: ```suggestion
Registry:
```

> : the verifiable append-only data structure that stores Claims in a transparency service. SCITT supports multiple Registry and Receipt formats to accommodate different transparency service implementations, such as historical Merkle Trees and sparse Merkle Trees.

Reviewer: ```suggestion
: a verifiable append-only data structure that stores Claims in a Transparency Service. SCITT supports multiple Registry and Receipt formats to accommodate different Transparency Service implementations, such as historical Merkle Trees and sparse Merkle Trees. Registry is a synonym to append-only log or ledger.
```

> Transparency Service (or Notary):

Reviewer: ```suggestion
Transparency Service:
```

> : the entity that maintains and extends the Registry, and endorses its state. A Transparency Service can be a complex distributed system, and SCITT requires the TS to provide many security guarantees about its Registry . The identity of a TS is captured by a public key that must be known by Verifiers in order to validate Receipts.

Reviewer: ```suggestion
: an entity that maintains and extends the Registry, and endorses its state. A Transparency Service is often referred to by its synonym Notary. A Transparency Service can be a complex distributed system, and SCITT requires the TS to provide many security guarantees about its Registry . The identity of a TS is captured by a public key that must be known by Verifiers in order to validate Receipts.
```

                |
                v                      +------------------+
 Issuer    -> Statement    Envelope    | DID Key Manifest |

>                 \           /          |  (decentralized)  |

Reviewer: ```suggestion
                \           /          |  (decentralized) |
```
